libs/utils/src/LabelColor.ts

```diff
@@ -0,0 +1,138 @@
+export enum LabelColorName {
```

Comment:
this whole file is great how it is all typescripted out properly 👏🏼

Reply:
🙇